.circleci/config.yml

```diff
@@ -165,6 +165,26 @@ jobs:
       - run: docker image rm ccdlstaging/dr_transcriptome
       - run: docker image rm ccdlstaging/dr_illumina
 
+  affy_agilent_tests:
+    machine: true
+    working_directory: ~/refinebio
+    steps:
+      - checkout
+
+      # Clean up stuff circle provides that we don't need.
+      - run: sudo bash .circleci/cleanup_instance.sh
+
+      # Setup Postgres in a Container
```

Comment:
Any special reason why `C` is capitalized?

Reply:
I guess because the comment is like a title which is why postgres is also capitalized? I just copied this line from the other job's config.